### DISCUSSION TITLE
[server not connecting][Hw5]

### DISCUSSION THREAD

**Discussion by rabberdabber** [prosyslab-classroom/cs348-information-security#745] [Q&A]
I couldn't connect to my server. It times out. Could you fix it soon? I didn't backup my code unfortunately.

**Comment by RiceBiscuits** [comment]
Please contact with KCloud administrator ASAP.

**Comment by rabberdabber** [comment]
ok, Thanks!

**Comment by rabberdabber** [comment]
Who do I contact? Sorry!

**Comment by Junseon00** [comment]
You can contact to kcloud@kaist.ac.kr

**Comment by rabberdabber** [comment]
Thanks!

**Comment by AnHaechan** [comment]
Has your problem resolved? I am having same problem but worktime is already over so..

**Comment by AnHaechan** [comment]
Ok, I emailed kcloudvpn@kaist.ac.kr and immediately resolved the problem. Even if the worktime is over, they helped me solving the issue. Those who have similar problems should send email.
